### PR TITLE
[benj.cloud] Add ruleset

### PR DESCRIPTION
**Pull request by bn4t**

### CODE REVIEW

File: src/chrome/content/rules/benj.cloud.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="benj.cloud">
+	<target host="benj.cloud" />
+  <target host="office.benj.cloud" />
```
Comment (J0WI) [inline src/chrome/content/rules/benj.cloud.xml]: Please make sure that you use only tabs to indent lines.

**Comment by J0WI** [comment]
The test is still failing. @Hainish do you know how to interpret «`not enough values to unpack (expected 2, got 1)`»?

**Comment by Hainish** [comment]
@J0WI the fetch script is trying to unpack a tuple which consists of an HTTP code as well as the HTML contents in https://github.com/EFForg/https-everywhere/blob/385e5e6d9a6e5bce13092fa6e1d99fa6a7ccf84b/test/rules/src/https_everywhere_checker/check_rules.py#L140

It looks like `office.benj.cloud` is open on port 80, but not responding to requests.  This is odd, and such an edge case is causing an exception.
```
(.venv3) user@https-everywhere ~/workspace/https-everywhere (master) $ curl -vv http://office.benj.cloud/
*   Trying 195.201.19.213...
* TCP_NODELAY set
* Connected to office.benj.cloud (195.201.19.213) port 80 (#0)
> GET / HTTP/1.1
> Host: office.benj.cloud
> User-Agent: curl/7.52.1
> Accept: */*
>
* Curl_http_done: called premature == 0
* Connection #0 to host office.benj.cloud left intact
```

**Comment by J0WI** [comment]
@bn4t could you whitelist this rule from fetch test [here](https://github.com/EFForg/https-everywhere/blob/master/utils/ruleset-whitelist.csv)? This seems to be a reasonable fix here.

**Comment by bn4t** [comment]
@J0WI How is the hash in the file you mentioned generated. What's the input data?

**Comment by J0WI** [comment]
It's just the SHA-256 of the file (in this case `benj.cloud.xml`). You can use `sha256sum` or any other tool to generate it.

**Comment by bn4t** [comment]
Still fails. Did I do something wrong when I added it to the whitelist? @J0WI

**Comment by J0WI** [comment]
I get the following result:
```
$ sha256sum benj.cloud.xml 
bd0efbba30261e92f11877fedb766e52efd5a9c7940b8db5db84c9096bb2ddcc  benj.cloud.xml

**Comment by bn4t** [comment]
Hmm. Weird. I'm currently at work so I generated it using [this](https://emn178.github.io/online-tools/sha256_checksum.html) tool. Going to try again using your result.

**Comment by J0WI** [comment]
Thanks!
